drivers/misc/ipts/companion.c

```diff
@@ -1,27 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ *
+ * Intel Precise Touch & Stylus
+ * Copyright (c) 2016 Intel Corporation
```

Comment:
Note: I added the intel copyright header to all of the files, even the ones that were added later on.

Having some files with the header, and some files without looked dumb to me somehow, and Intel provided the biggest part of the driver and the technology. As far as "my" files are concerned (I *think* I am the only one who ever added files tbh.), they are mostly just other code from Intel moved around anyways.

Of course we could and should add other names there if we decide to. Or maybe just add an entry for `linux-surface` as whole.

But I just wanted to mention it in case someone got confused about why I added those.

Reply:
I'm pretty sure you could pin your name at least on the companion driver. I mean you pretty much wrote every part of it. But we can always add that later.